ta4j-core/src/main/java/org/ta4j/core/criteria/VersusEnterAndHoldCriterion.java

```diff
@@ -35,7 +35,23 @@
  *
  * <p>
  * Compares the value of a provided {@link AnalysisCriterion criterion} with the
- * value of an {@link EnterAndHoldCriterion}.
+ * value of an {@link EnterAndHoldCriterion} by using the following formula:
+ *
+ * <pre>
+ * xVersusEnterAndHold = (x - enterAndHold) / (abs(enterAndHold)
+ * </pre>
+ *
+ * <p>
+ * It can be used for absolute or percentage criteria values, for example:
+ *
+ * <ul>
+ * <lI>ProfitVsEnterAndHold = (400$ - 500$) / abs(500$) = -0.2 (= the trading
+ * profit is 20% worse than the profit that would be made by simply entering and
+ * holding)
+ * <li>ProfitLossPercentageVsEnterAndHold = (0.9 - (-0.3)) / abs(-0.3) = +4 (=
```

Comment:
Why is 20% less than enter-and-hold represented by -0.2? Shouldn't it be 0.8? 

This is exactly why the same class producing semantically different values via a `addBase` flag introduces needless complexity and defect potential. That the examples in this javadoc mix them up says it all.

Any solution needs to include splitting the `ReturnCriterion` into separate classes.